sklearn/tree/_tree.pxd

```diff
@@ -100,6 +100,5 @@ cdef class TreeBuilder:
     cdef double min_impurity_decrease   # Impurity threshold for early stopping
 
     cpdef build(self, Tree tree, object X, np.ndarray y,
-                np.ndarray sample_weight=*,
-                np.ndarray X_idx_sorted=*)
+                np.ndarray sample_weight=*)
```

Comment:
(This is valid cython? Interesting)

Reply:
Do you mean the use `=*`?

Reply:
Yea, I can't find in the Cython doc what does this. I guess it means "there is a default keyword argument".

Reply:
Yep, you are absolutely right. Just in case you wanna have a look, here is the [documentation](https://cython.readthedocs.io/en/latest/src/userguide/language_basics.html#optional-arguments).